### ISSUE TITLE
CV605

### ISSUE THREAD

**Issue by paule26** [bitfocus/companion-module-marshall-ipcamera#12]
Could you please add the CV605
The module works perfect with my CV730, but no connection to the CV605.
I thougt you just forget to mention the CV605 on your list, so I bought one, no good idea ...

**Comment by whytheheckme** [comment]
Agreed, this would be very helpful. Any insight on how to make this happen?

**Comment by paule26** [comment]
I now control all my Cameras with the Sony Visca Module, works perfect and is absolutely stable in opposit to this Module

**Comment by coleinmb** [comment]
How did you get the Visca module to work with this cam?
